### PR TITLE
DOCSUP-14786: Executable and ExecutablePool table engines, executable() table function, settings

### PR DESCRIPTION
**Pull request by olgarev**
Changelog category:
- Documentation for #28518, #28102, #28833

### CODE REVIEW

File: docs/en/engines/table-engines/special/executable.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
+---
+toc_priority: 47
+toc_title: Executable/ExecutablePool
+---
+
+# Executable and ExecutablePool Table Engines {#table_engines-executable}
+
+`Executed` and `ExecutablePool` engines allow executing a custom script and passing data into this script. This data comes as a result of queries, executed in ClickHouse.
```
Comment (olgarev) [inline docs/en/engines/table-engines/special/executable.md:8]:
```suggestion
`Executable` and `ExecutablePool` engines allow executing a custom script and passing data into this script. This data comes as a result of queries, executed in ClickHouse. 
```

File: docs/en/operations/settings/settings.md [hunk 2]
```diff
@@ -3832,6 +3833,32 @@ Default value: `0`.
...
+
+-   Positive integer.
+-   0 — No time restrictions.
+
+Default value: `0`.
+
+## command_termination_timeout {#command_termination_timeout}
+
+After a script of the [Executable/ExecutablePool](../../engines/table-engines/special/executable.md) table engine is executed, the engine has `command_termination_timeout` seconds to shutdown, before ClickHouse throws an exception. Specified in seconds. 
+=======
```
Comment (olgarev) [inline docs/en/operations/settings/settings.md:3861]:
```suggestion
```

File: docs/en/operations/settings/settings.md [hunk 3]
```diff
@@ -3871,6 +3898,18 @@ The maximum number of threads for background data parsing and insertion.
...
+Default value: `0`.
+
+## send_chunk_header {#send_chunk_header}
+
+Defines the number of lines, ended with '\n', which are sent before sending chunk to process of the [Executable/ExecutablePool](../../engines/table-engines/special/executable.md) table engines. 
+
+Possible values:
+
+-   Positive integer or zero.
+=======
```
Comment (olgarev) [inline docs/en/operations/settings/settings.md:3912]:
```suggestion
```

File: docs/en/operations/settings/settings.md [hunk 4]
```diff
@@ -3871,6 +3897,17 @@ The maximum number of threads for background data parsing and insertion.
...
+
+Default value: `0`.
+
+## send_chunk_header {#send_chunk_header}
+
+Defines the number of lines, ended with '\n', which are sent before sending chunk to process of the [Executable/ExecutablePool](../../engines/table-engines/special/executable.md) table engines. 
+
+Possible values:
+
+-   Positive integer or zero.
```
Comment (olgarev) [inline docs/en/operations/settings/settings.md:3910]:
```suggestion
-   Positive integer.
```

**Comment by mzealey** [comment]
This does not seem to have been merged, even though the functionality is in clickhouse?

**Comment by kitaisreal** [comment]
@mzealey implementation in that time was not completely finished to be documented, because at the same time new features were constantly added to similar functionality executable UDF, executable dictionaries, and documentation should be updated with use cases specific to executable storage. Now it is finished, documentation will be updated.

**Pull request closed without merging**